src/components/markdown/Preview.js

```diff
@@ -1,15 +1,14 @@
 import React from 'react';
-import PropTypes from 'prop-types';
 import marked from 'marked';
 import styles from 'github-markdown-css/github-markdown.css';
+import { useSelector } from '../../hooks/markDownContext';
+import { getMarkDowns } from '../../selectors/markDownSelectors';
 
-const Preview = ({ markdown }) => {
-  const __html = marked(markdown);
-  return <div className={styles['markdown-body']} dangerouslySetInnerHTML={{ __html }}></div>;
-};
+const Preview = () => {
 
-Preview.propTypes = {
-  markdown: PropTypes.string.isRequired
+  const state = useSelector(getMarkDowns);
+  const __html = marked(state);
```

Comment:
create another selector `getMarkDownsPreview` that uses `marked` to get your markdown preview html